psqlgraph/node.py

```diff
@@ -209,7 +210,8 @@ def __init__(self, node_id=None, properties=None, acl=None,
         self._props = {}
         self.system_annotations = system_annotations or {}
         self.acl = acl or []
-        self.properties = properties or {}
+        self.properties = sanitize(getattr(self, "_defaults", {}))
```

Comment:
`sanitize` is called within the setter for `properties` so you do not have to call it again. Also I do think having the property defined explicitly here would make sense